lib/dolly/document.rb

```diff
@@ -51,7 +51,8 @@ def rev= value
       doc['_rev'] = value
     end
 
-    def save
+    def save options = {}
+      return false unless options[:validate] == false || valid?
```

Comment:
@rubenrails @netlogics Agree on conventions, just trying to keep the focus. Because "implemting" have a meaning... On this particular PR I don't see the harm